test/e2e/nodelatencymonitor_test.go

```diff
@@ -0,0 +1,194 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	v1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	statsv1alpha1 "antrea.io/antrea/pkg/apis/stats/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	nodeLatencyMonitorName = "default"
+	nodeLatencyMonitorNS   = "kube-system"
+	pingIntervalSeconds    = 50
+)
+
+func TestNodeLatencyMonitor(t *testing.T) {
+	skipIfNotRequired(t)
```

Comment:
I am not sure why you included this? With a single argument, I believe this is a no-op

Reply:
however, the test should be skipped if the feature (`NodeLatencyMonitor`) is disabled. We have the `skipIfFeatureDisabled` helper function for this.